cfg/rippled-example.cfg

```diff
@@ -1248,6 +1248,30 @@
 #       Example:
 #           owner_reserve = 2000000      # 2 XRP
 #
+#   extension_compute_limit = <gas>
```

Comment:
These configs should not be here, right? These should be hardcoded and be changed only by fee voting or an amendment.

Reply:
The way you vote (and the way you set values in standalone mode) is via the config